.fixtures.yml

```diff
@@ -2,6 +2,9 @@ fixtures:
   repositories:
     stdlib:     'https://github.com/puppetlabs/puppetlabs-stdlib.git'
     extlib:     'https://github.com/voxpupuli/puppet-extlib'
+    certs:
+      repo: 'https://github.com/ehelms/puppet-certs'
+      ref: '3bf95e54f1e5182d39c47b78533c1ef99e57e6e5'
```

Comment:
This is temporary until https://github.com/theforeman/puppet-certs/pull/332 is available

Reply:
You can also use `branch` which makes it easier to push an update and just rerun the tests.